test/emdawnwebgpu_link_test.cpp

```diff
@@ -0,0 +1,235 @@
+// Copyright 2021 The Emscripten Authors.  All rights reserved.
```

Comment:
Can you name this file starting with `test_` to match the name of the test in python?